internal/registry/registry.go

```diff
@@ -91,6 +92,19 @@ func (r *registry) VerifyModuleExists(layer v1.Layer, pathPrefix, kernelVersion,
 	return err == nil
 }
 
+func (r *registry) GetDigest(ctx context.Context, image string, tlsOptions *kmmv1beta1.TLSOptions, registryAuthGetter auth.RegistryAuthGetter) (string, error) {
+	pullConfig, err := r.getPullOptions(ctx, image, tlsOptions, registryAuthGetter)
+	if err != nil {
+		return "", fmt.Errorf("failed to get pull options for image %s: %w", image, err)
+	}
+
+	digest, err := crane.Digest(image, pullConfig.authOptions...)
+	if err != nil {
+		return "", fmt.Errorf("failed to get digest for image %s: %w", image, err)
```

Comment:
Do we really need `%w` here?
```suggestion
		return "", fmt.Errorf("failed to get digest for image %s: %v", image, err)
```

Reply:
Hm, you're probably right, we currently don't and most probably won't leverage the error wrapping for the callers of this API. I was just following the rest of this API's source code to be consistent. Should I change only these occurrences I've added here?

Reply:
I think so. I would not make unrelated changes to the existing API in this PR.